### ISSUE TITLE
[Bug]: lost connection during execution

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#6329]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 92439b1d3f73552ca912de6180099821c39955fd
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

during running a SQL file of several DDL sqls, the following error occurred on the client side and strange errors on the server side.  All logs can be found here: 
[multi-cn.log.tar.gz](https://github.com/matrixorigin/matrixone/files/9946003/multi-cn.log.tar.gz)

<img width="758" alt="image" src="https://user-images.githubusercontent.com/77312370/200175356-5463f529-daa8-4816-bd19-e16ee968747c.png">

### Steps to Reproduce

1. start a cluster with two CNs(If you want to start a mo cluster in eks, you can following this guideline document: https://github.com/matrixorigin/docs/blob/main/dev/guide/MO%20Operator%20%E7%9A%84%E4%BD%BF%E7%94%A8%E4%B8%8E%E6%B5%8B%E8%AF%95%EF%BC%88%E5%86%85%E9%83%A8%E7%89%88%E6%9C%AC%EF%BC%89.md)
2. execute `drop table if exist` `create table xxx` repeatedly

### Additional information

Configuration of each node is here: 
[Archive.zip](https://github.com/matrixorigin/matrixone/files/9946010/Archive.zip)

**Comment by daviszhen** [comment]
I do not reproduce the issue using following eks configuration.
[mo.yaml.zip](https://github.com/matrixorigin/matrixone/files/9949197/mo.yaml.zip)

**Comment by daviszhen** [comment]
I have reproduced in the @sukki37 's eks cluster. I need add debug info and start the cluster outside the eks cluster.

**Comment by daviszhen** [comment]
After syncing with @sukki37 and adding S3 as one feature, I reproduced the case in the eks cluster. The result denoted that the CN service's RPC may take the responsibility. The RPC consumed all the time(10minutes) that came from the context's timeout(10minutes).  
So, The CN people should take over the case.

The repoduced results:

<img width="1462" alt="hang" src="https://user-images.githubusercontent.com/60595215/200333063-c541a67b-4af1-4746-a0ab-df7a3ba18c67.png">

**Comment by fengttt** [comment]
@zhangxu19830126 verified this can be reproed with dev single process, 2cn-1dn setting.

**Comment by nnsgmsone** [comment]
Fixed the problem, but not yet verified. Zhang Xu and Peng Zheng will help verify again.

**Comment by nnsgmsone** [comment]
It should be fixed by #6408. Please ask Peng Zheng to help to cross-check again.

**Comment by nnsgmsone** [comment]
I was helping Yuanning check out tpcc #6335. so I didn't look at this issue for now.

**Comment by JackTan25** [comment]
I'm trying to repro this one, not sure it's solved by someone?

**Comment by fengttt** [comment]
Loop 10 times.   If all pass, close.

**Comment by jensenojs** [comment]
> @JackTan25 Loop 10 times. If all pass, close.

Because the reproduction of several different issues is the same., we created a set of tests that contained nearly three thousand sets of “drop table if exists and create table”. There is no problem on the 2cn-1dn. It may take 30, 000 tests to run for 10 minutes. But on K8s, there is an occasional problem that a statement takes a long time to execute.

We will further confirm that there is no problem, and then hand it over to the test team after Zhang Xu's pr merger.

![2b7dc1d76f38e3023f8315c8229117d](https://user-images.githubusercontent.com/56761542/201017605-37a2a1a0-cd1b-4811-8f4c-1a8cc61f4095.jpg)

**Comment by JackTan25** [comment]
Now I have Tried to find rpc timeout log, for now, it doesn't appear. Let me continue. My envirment is 2cn,1dn,3 logservice, use eks cluster.

**Comment by JackTan25** [comment]
I tried so many times, but for now, it will result in these. That's the rpc bug, time out. We need to wait zhangxu
[start_up_failed_cn0.txt](https://github.com/matrixorigin/matrixone/files/9985971/start_up_failed_cn0.txt)
[startup_failed_cn1.txt](https://github.com/matrixorigin/matrixone/files/9985972/startup_failed_cn1.txt)
let me continue to test.

**Comment by aressu1985** [comment]
in progress

**Comment by aressu1985** [comment]
now mutil-cn mo is still not stability, thes types of issues will be regressed till the BVT in mutil-cn mo can pass

**Comment by aressu1985** [comment]
fixed: 0.6.0-1786399（image-tag）

today has run bvt on eks with 2-cn for 3 times, and this problem not happen again
